lib/cogctl/actions/relay_groups/util.ex

```diff
@@ -16,17 +16,25 @@ defmodule Cogctl.Actions.RelayGroups.Util do
     relay_message = Enum.join(relay_messages, "\n")
     Table.format(group_info, sort) <> "\n\n" <> relay_message
     |> display_output
-    if error_count > 0, do: :error
+    if error_count > 0 do
```

Comment:
No, error count is the number of errors that we may have had when assigning a relay to a relay group. The count tells me how many failures. If the number is greater than 0, then I want to return :error to show that there was a problem.